integration-cli/docker_utils.go

```diff
@@ -267,7 +267,7 @@ func (d *Daemon) Cmd(name string, arg ...string) (string, error) {
 
 func daemonHost() string {
 	daemonUrlStr := "unix:///var/run/docker.sock"
-	if daemonHostVar := os.Getenv("DOCKER_TEST_HOST"); daemonHostVar != "" {
```

Comment:
ok, sounds good then. I shall not make future references to DOCKER_TEST_HOST :yum: